src/advanced/configuration-updater.md

```diff
@@ -115,6 +120,7 @@ config][stored_configuration] serialization. It has the following fields:
 - **services**: Object  
   Service-specific configuration parameters.
 
+[ta-config]: https://docs.rs/exonum/0.4.0/exonum/blockchain/config/enum.TimeoutAdjusterConfig.html
```

Comment:
Very minor nit: could you move this line to the end of the file? I think, having all links at the same place (if there are not lots of them) is more manageable than having them scattered throughout the text.